FWCore/Utilities/interface/IndexSet.h

```diff
@@ -0,0 +1,42 @@
+#ifndef FWCore_Utilities_IndexSet_h
+#define FWCore_Utilities_IndexSet_h
```

Comment:
@makortel sorry I wasn't clear. The class should stay in FWCore/Utilities. If later it is decided the type should be stored, then at that time it would have to move to DataFormats/Common.